config/routes.rb

```diff
@@ -5,6 +5,8 @@
       namespace :merchants do
         get '/find', to: 'search#show'
         get '/find_all', to: 'search#index'
+        get '/most_revenue', to: 'revenue#index'
+        get '/most_items', to: 'items#index'
```

Comment:
These routes seem to be more restful as they are routing to restful actions, originally set this up as a business controller but it was getting ugly so pulled out the logic into separate controllers